### ISSUE TITLE
Repeater/Context Menu: Separator in menu as list item (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#863]
### Expected behavior

Within the context menu, the separator element is a horizontal rule and not seen as a list item in the menu. 
### Actual behavior

Separator is a list item (`<li>`) and interpreted as an item in list by screenreaders.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.12.0](https://github.com/blackbaud/skyux/releases/tag/v1.12.0)
